src/printapp/templates/index.html

```diff
@@ -4,69 +4,64 @@
   
   <script type="text/x-handlebars" data-template-name="application">
   {{#if isAuthenticated}}
-    {{partial "welcome"}}
+	{{partial "welcome"}}
```

Comment:
Run it through your own formatter.

--- Original Message ---

From: "James Lamine" <notifications@github.com>
Sent: May 8, 2015 5:40 PM
To: "jglamine/calvinwebprint" <calvinwebprint@noreply.github.com>
Cc: "FreedCapybara" <templeandrew@hotmail.com>
Subject: Re: [calvinwebprint] Design (#46)

> @@ -4,69 +4,64 @@
>
>    <script type="text/x-handlebars" data-template-name="application">
>    {{#if isAuthenticated}}
> -    {{partial "welcome"}}
> +     {{partial "welcome"}}

This need to be less indented

---
Reply to this email directly or view it on GitHub:
https://github.com/jglamine/calvinwebprint/pull/46/files#r29979224